### ISSUE TITLE
question about essay model

### ISSUE THREAD

**Issue by ewwll** [leoxiaobin/deep-high-resolution-net.pytorch#242]
Hello, I used the network model in this article to estimate the gaze point, but the loss function is already very small during the training process, but there is no output heat map but the output is more and more like the contour of the original input picture. How is this going
